darts/timeseries.py

```diff
@@ -3199,6 +3228,18 @@ def plot(
                 )
             )
 
+        if isinstance(label, list):
```

Comment:
Checking for an actual list might be a little restrictive here. For instance it'd make sense to accept tuples as well. Perhaps check for `Sequence` type instead (it's usually a good fit if you rely on the order and the length being well defined).